index.bs

```diff
@@ -4153,15 +4153,25 @@ Requirements for Transition</h5>
 	and <em class=rfc2119>must not</em> include any such marking
 	if not already present.
 
+	If the document's most recent publication is a [=Candidate Recommendation Draft=],
+	the [=Team=] <em class=rfc2119>must</em> verify
+	that it contains no changes since the previous [=Candidate Recommendation Snapshot=] other than:
+	* [=editorial changes=],
+		including the addition or removal of a statement as to whether the document will [=allow new features=]
```

Comment:
tl;dr: hard to express discomfort, no proposal to change the PR wording here.

I have a feeling that adding or removing such a statement is more substantive a change than it might at first appear, but I'm having trouble expressing why. I can't think of any patent implications though, so I guess this is okay.

My concern is to do with this having a significant potential long term impact on the way the report is perceived, and how it can be amended over time, that warrants some care, but I guess it's highly unlikely that AC would ever want to overturn a WG decision about it. As long as the WG's working mode doesn't allow changes like this to be introduced in CRDs without a WG Decision, it's okay...

Reply:
Wrt the edits to the Process, I think we should just pull it out into its own bullet in this list, separate from [=editorial changes=].

But also, I don't think we're clear anywhere that this is not something that the WG can change out from under the AC during its review?

Reply:
Maybe we can clarify that changing from allowing to disallowing new features is a [=substantive change=].

Reply:
I've added https://github.com/w3c/process/pull/1070/commits/f9c1c89aab1362434ea4cda0a14f16e65233c413 to address this concern.

Reply:
That's definitely an improvement, thanks @frivoal if for no other reason than it avoids a potential misunderstanding of the combination of negative + includes. Not sure it addresses the point about changing from under the AC during its review though?

Reply:
> Adding or removing this allowance is a [=substantive change=].

does that, since you cannot silently make substantive changes post AC review without another round of actively checking for consensus.

Reply:
The review is about a single dated document. If changes and new publications are made during or after the review, they aren't on the same dated revision of the document, and they are irrelevant to that review, unless the provisions of https://www.w3.org/policies/process/drafts/#ACReviewAfter are used to apply additional changes to the reviewed material.